airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -35,6 +35,7 @@
     DestinationType.SNOWFLAKE.value: 255,
     DestinationType.REDSHIFT.value: 127,
     DestinationType.POSTGRES.value: 63,
+    DestinationType.ORACLE.value: 30,
```

Comment:
that's sooo short... 😨

Reply:
@ChristopheDuong do you think this is a problem?

Reply:
Since we reserve 12 characters for dbt and 8 for airbyte, that leaves you table names of 10 characters max... we should probably make sure to show how to increase the size limit for oracle in docs and tests... otherwise, output table names from normalization are going to be so un-useable...  

(in that case, I would not recommend supporting normalization for oracle with only 10 characters?)